Content.Shared/Access/Components/IdCardComponent.cs

```diff
@@ -21,4 +18,17 @@ public class IdCardComponent : Component
         [DataField("jobTitle")]
         public string? JobTitle;
     }
+
+    [Serializable, NetSerializable]
+    public sealed class IdCardComponentState : ComponentState
+    {
+        public string? FullName;
+        public string? JobTitle;
+
+        public IdCardComponentState(string? fullName, string? jobTitle)
+        {
+            FullName = fullName;
+            JobTitle = jobTitle;
+        }
+    }
```

Comment:
i dont see any feasible way to avoid telling the client the names of any entities at all

Reply:
only way to do that would be to -literally- change the name of the entity

Reply:
Yeah that's probably not ideal. The server should still have the correct metadata name and there's no easy way to override it for clients that doesn't involve sending the data to the client first. Could bikeshed it later if someone thinks of a better way to do it. We already hide uplinks at least so that's something.

Reply:
Alright, this is fine, we can figure it out later.